线框图变得不那么重要了 — 好事啊！ (#5357)
xitu/gold-miner: TODO1/wireframes-are-becoming-less-relevant-and-thats-a-good-thing.md

> * 译者：
> * 校对者：

# 线框图变得不那么重要了 — 好事啊！

![](https://cdn-images-1.medium.com/max/3160/1*-uTCxjdcMtjgKnKfUJqixA.png)

> 随着时间的推移，我发现线框图越来越没用，而且不止我一个人如此认为，因为关于该术语的定义太过松散，所以具体的定义可能很重要，虽然有许多原型可以检测不同维度的逼真度，但我想重点介绍在听到“线框”一词时最能想到的特定变化，它不是一个草图或一个实成的原型，而是一个典型的中间状态(数字艺术设计中故意不加样式，并用黑白表示整个页面的“骨架”)。原型线框图试图准确的表示布局和信息框架，并**有意的**避免高逼真的视觉呈现，但有时候这也避免了内容的高保真度。

Reviewer: 『虽然有许多原型可以检测不同维度的逼真度』 => 『虽然有许多原型可以检测不同[维度](https://medium.com/in-the-hudl/prototype-fidelity-its-more-than-high-and-low-4dedb4cb1a0)的逼真度』

![Behold the drab, spindly splendor of a wireframe in it’s natural habitat](https://cdn-images-1.medium.com/max/3360/0*0FGSCBVTYpA2Dt4z.png)

在我参与的线下或线上讨论中，我惊讶的发现线框图依然被定义为设计环节中的**基本**步骤，虽然这种态度似乎处于下降趋势，但我仍然能听到每个人 — 从早期的职业设计师到业界领先的机构，都在坚持“线框阶段”的必要性，这些争论通常有如下几点：

> - 线框图关注于实用性而不是艺术性，它能阻止利益相关者在一些无关紧要的细节(如：按钮颜色)讨论上破坏会议，并允许使用者在测试时集中于交互而不是视觉呈现。
>

> > - 线框图的创建更快，它能保持设计理念，并避免了过多的开销或过于依赖于某一特定方向的风险。

Reviewer: 『 并避免了过多的开销或过于依赖于某一特定方向的风险。』=> 『并避免在某些无关紧要的细节投入过多的精力。』

> > - 线框图的创建更快，它能保持设计理念，并避免了过多的开销或过于依赖于某一特定方向的风险。

Author: > 『 并避免了过多的开销或过于依赖于某一特定方向的风险。』=> 『并避免在某些无关紧要的细节投入过多的精力。』

这里我的理解还是不能把原文的词语给替代掉

>
> - 线框图作为一种没有额外视觉设计开销的详细交互文档工具，也存在这一些分歧(更多面向企业)。

> 实际上，并不是说每个人都在制作线框图，但当有人不承认时，他通常会保持沉默，并不敢正眼看你，虽然他们想让更多的人加入他们，不过他们往往受限于他们所处的组织、利益相关者或项目本身。但他们的心态是必要的，关于它们优势的信念被误导了，虽然我不否认线框图的价值，但现在它只在有限情况下才有价值，行业思维和实践的大量变动正促成这种改变，这是值得深思的。

Reviewer: 『但当有人不承认时，他通常会保持沉默，并不敢正眼看你』 => 『但是当有人承认不做线框图，他总会躲避你的目光并且小声的说』

> 实际上，并不是说每个人都在制作线框图，但当有人不承认时，他通常会保持沉默，并不敢正眼看你，虽然他们想让更多的人加入他们，不过他们往往受限于他们所处的组织、利益相关者或项目本身。但他们的心态是必要的，关于它们优势的信念被误导了，虽然我不否认线框图的价值，但现在它只在有限情况下才有价值，行业思维和实践的大量变动正促成这种改变，这是值得深思的。

Author: > 然后很多标点都错了
> 
> 句号都变成了逗号

标点符号是我优化了的，我觉得有些地方不应该用句号，因为语句还没结束。

## 不断变换的产品开发方法正改变着设计的完成方式

**敏捷**产品的开发鼓励更少的线性过程，不是从整个产品的基础元素开始，然后在基础元素上构建层，而是将重点转移到更小，更频繁地交付完全实现的“垂直”切片。

![](https://cdn-images-1.medium.com/max/2000/1*ZP-WJyxl2cwFlE_SifUCkA.png)

这也包括完全实现的设计，这使得同时考虑可视化、信息架构和交互性变得非常重要，在敏捷开发团队中，你的下一次迭代将在几天或几周内进行，而不是几月或几年，这对会产生广泛的前期绘图阶段的线框设计来说，很难有一席之位。

> [**精益UX(Lean UX)**](https://www.smashingmagazine.com/2011/03/lean-ux-getting-out-of-the-deliverables-business/)是一种随之诞生的改进流程(似乎在不断被关注)，在Lean设计中，繁重类的设计工作被忽略或完全省略，这有利于在产品本身上进行协作，Lean设计还反驳了将线框图作为文档工具的想法，它建议实际产品的屏幕布局可以作为它们的文档，你自己创建的设计组件将成为临时项 —— 只要它们包含的信息在实际产品中以可访问的方式存在，就可以把他们放到一边去。其它相关情况仍可记录在案，但仅在设计时或之后。

Reviewer: ```suggestion
[**精益UX(Lean UX)**](https://www.smashingmagazine.com/2011/03/lean-ux-getting-out-of-the-deliverables-business/)是一种随之诞生的改进流程(似乎在不断被关注)，在 Lean 设计中，繁重类的设计工作被忽略或完全省略，这有利于在产品本身上进行协作，Lean 设计还反驳了将线框图作为文档工具的想法，它建议实际产品的屏幕布局可以作为它们的文档，你自己创建的设计组件将成为临时项 —— 只要它们包含的信息在实际产品中以可访问的方式存在，就可以把他们放到一边去。其它相关情况仍可记录在案，但仅在设计时或之后。
```

### 可用性和信息框架不应被漠视

风险是显而易见的 —— 界面中元素的可视化表示会影响到它们的感知方式。类似于：表单元素的下拉菜单可能会让人误解其功能，与菜单条目不同的菜单标题可能会被忽视。颜色和对比度在引导视觉注意方面起着巨大的作用。如果你在一个线框上做可用性测试，当你最终改变所有的外观时，将会丢失很多内容。为了更好地适应视觉效果，您可能还必须在这一点上进行更多的布局更改。用户可能只会专注于视觉元素，他们这样做无可厚非，而且将所有内容设置为灰色并不一定会降低这种风险。关于可用性或信息框架的问题通过理解问题和测试任务的完成比口头评论更好地暴露出来。只要你能很好的设定预期并注意反馈的征求，就可以用任何保真度来实现概念的输入。

### 线框图对利益相关者影响很少

> 我懂了，在会议中让利益相关者专注于按钮或副本的颜色可能会令人难以置信地沮丧。但是，通过有意避免更高逼真度的视觉效果，并把lorem ipsum(译者注: 排版时的专用测试文字)随意放置，你真的能解决问题吗？还是你只是推迟了问题的解决？你喜欢低保真度是因为它让人们更容易理解和输入信息吗？或者事实是：它使别人**更难**理解，我们**喜欢**这样，因为它让我们能够将自己的专业知识凌驾于他人之上，免受批评？如果利益相关者对IA的评论不感兴趣，而是更专注于接近他们专业知识的其它事情，那你为何还要斗争呢？你才是信息架构方面的专家，而不是他们，任何需要对布局进行的更改都应通过对用户的测试而变得清晰。

Reviewer 2: ```suggestion
我懂了，在会议中让利益相关者专注于按钮或副本的颜色可能会令人难以置信地沮丧。但是，通过有意避免更高逼真度的视觉效果，并把 lorem ipsum （译者注: 排版时的专用测试文字）随意放置，你真的能解决问题吗？还是你只是推迟了问题的解决？你喜欢低保真度是因为它让人们更容易理解和输入信息吗？或者事实是：它使别人**更难**理解，我们**喜欢**这样，因为它让我们能够将自己的专业知识凌驾于他人之上，免受批评？如果利益相关者不热衷于评价信息架构，而是更在意他们专业知识相关的其它事情，那你为何还要斗争呢？你才是信息架构方面的专家，而不是他们，任何需要对布局进行的更改都应通过对用户的测试而变得清晰。
```

> 我懂了，在会议中让利益相关者专注于按钮或副本的颜色可能会令人难以置信地沮丧。但是，通过有意避免更高逼真度的视觉效果，并把lorem ipsum(译者注: 排版时的专用测试文字)随意放置，你真的能解决问题吗？还是你只是推迟了问题的解决？你喜欢低保真度是因为它让人们更容易理解和输入信息吗？或者事实是：它使别人**更难**理解，我们**喜欢**这样，因为它让我们能够将自己的专业知识凌驾于他人之上，免受批评？如果利益相关者对IA的评论不感兴趣，而是更专注于接近他们专业知识的其它事情，那你为何还要斗争呢？你才是信息架构方面的专家，而不是他们，任何需要对布局进行的更改都应通过对用户的测试而变得清晰。

Reviewer: ```suggestion
我懂了，在会议中让利益相关者专注于按钮或副本的颜色可能会令人难以置信地沮丧。但是，通过有意避免更高逼真度的视觉效果，并把lorem ipsum(译者注: 排版时的专用测试文字)随意放置，你真的能解决问题吗？还是你只是推迟了问题的解决？你喜欢低保真度是因为它让人们更容易理解和输入信息吗？或者事实是：它使别人**更难**理解，我们**喜欢**这样是因为它让我们能够将自己的专业知识凌驾于他人之上，免受批评？如果利益相关者对 IA 的评论不感兴趣，而是更专注于接近他们专业知识的其它事情，那你为何还要斗争呢？你才是信息架构方面的专家，而不是他们，任何需要对布局进行的更改都应通过对用户的测试而变得清晰。
```

### 它变得更快捷，也更容易接近高保真的视觉呈现

几年前，Photoshop是UI设计之王，每个人都在争先恐后地看谁能在给定的页面上添加最详细、最逼真的纹理。当时有一个关于线框节省时间优势的论据，但今天，我们的设计工具专为UI设计(想想Sketch，Adobe XD，或Figma)而打造，它们在全局设计上更快，而且更容易的管理和更新样式。在设置好视觉风格后再改变布局不应该比以前改变它更难。对于那些成熟的设计组织来说，设计系统和支持这些系统的组件库的结合几乎消除了任何借口。即使还不是一个成熟的组织，你仍然可以访问与开发人员相匹配的UI框架的UI工具包。这也有助于：即使是最复杂的视觉美学，今天仍然是相对简单的。如果你的按钮只是一个带有圆角的蓝色长方形，那么你通过故意把它变成灰色真的节省了多少时间？

### 线框图还有意义吗？

> 它在合适的情况下是有意义的，你可以在以下情况下创建线框图：

Reviewer: ```suggestion
它在合适的情况下是有意义的。例如，你可以在以下情况下创建线框图：
```

- 你确实拥有一个视觉上很复杂的产品(如移动游戏)，并且需要独立的不可避免的漫长艺术创作来完成交互，即使如此，你仍然可以尽力实现风格的相似度。

- 你将它们用作练习，以帮助人们独立的了解信息框架(希望是产品开发的一次性而非反复性的部分)

> - 你想绘制或测试信息框架，但又依赖于他人的视觉设计(不爽)，或者在某些方面受限于你可用工具的视觉功能抑或你个人的技能组合。

Reviewer: ```suggestion
- 你想绘制或测试信息框架，但又依赖于他人的视觉设计(这并不理想！)，或者在某些方面受限于你可用工具的视觉功能抑或你个人的技能。
```

- 你处于过时的瀑布流或代理环境中，并且你对流程没有太多的自主权，这不是很好，但它超出了你的控制。

> 我确信还要很多其它的可能场景和例外情况，但我认为对于今天的大多数设计师来说可能并不常见，如果你认为传统的线框设计只是在真正适合问题的情况下才使用的方案，那么你会发现它们经常可以避免 — and that’s OK。

Reviewer: ```suggestion
我确信还有很多其它的可能场景和例外情况，但我认为对于今天的大多数设计师来说可能并不常见，如果你认为传统的线框设计只是在真正适合问题的情况下才使用的方案，那么你会发现它们经常可以避免 — and that’s OK.
```
